packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -368,13 +368,16 @@ async function handler(req: CustomRequest) {
   await Promise.all(promises);
 
   const workflows = await getAllWorkflowsFromEventType(bookingToDelete.eventType, bookingToDelete.userId);
+  const parsedMetadata = bookingMetadataSchema.safeParse(bookingToDelete.metadata || {});
 
   await sendCancelledReminders({
     workflows,
     smsReminderNumber: bookingToDelete.smsReminderNumber,
     evt: {
       ...evt,
-      metadata: { videoCallUrl: bookingMetadataSchema.parse(bookingToDelete.metadata || {})?.videoCallUrl },
+      ...(parsedMetadata.success && parsedMetadata.data?.videoCallUrl
```

Comment:
if it's success, you shouldn't need `data?.`?

Reply:
We need this because it would also succeed for `{}`